packages/card/src/card.css

```diff
@@ -44,19 +37,15 @@ slot[name='description'] {
     );
 }
 
-#preview + #cover-photo {
-    display: none;
-}
-
 #cover-photo ::slotted(*),
-:host(:not([variant='quiet'])) #preview ::slotted(*) {
-    width: 100%;
+#preview ::slotted(*) {
+    inline-size: 100%;
```

Comment:
Should include the `quiet` variant [as per spectrum-css](https://github.com/adobe/spectrum-web-components/blob/5cf5d34645bf9494ebd20f64c42d1619523d2d84/packages/card/src/spectrum-card.css#L567).